mk/nvidia-modprobe.mk

```diff
@@ -41,7 +41,10 @@ $(LIB_SRCS): $(SRCS_DIR)/.download_stamp
 
 .PHONY: all install clean
 
-all: $(LIB_STATIC)($(LIB_OBJS))
+all: $(LIB_STATIC)
```

Comment:
Can you explain this why this is needed :) ?

Reply:
you're basically running into this: https://www.gnu.org/software/make/manual/make.html#Archive-Pitfalls

GNU `make` will run parallel instantiations of `ar` to add each single `.o` to the `.a.`, leading to corruption of the archive.

By replacing `all: $(LIB_STATIC)($(LIB_OBJS))` with

```
all: $(LIB_STATIC)

$(LIB_STATIC): $(LIB_OBJS)
	$(AR) rs $@ $^
```

I turn this into a single call to `ar`, avoiding the race condition.